contracts/AnchoredView/AnchoredView.sol

```diff
@@ -0,0 +1,278 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+import "./SymbolConfiguration.sol";
+import "../OpenOraclePriceData.sol";
+
+interface CErc20 {
+    function underlying() external view returns (address);
+}
+
+interface AnchorOracle {
+    function numBlocksPerPeriod() external view returns (uint); // approximately 1 hour: 60 seconds/minute * 60 minutes/hour * 1 block/15 seconds
+
+    function assetPrices(address asset) external view returns (uint);
+
+
+    /* struct Anchor { */
+    /*     // floor(block.number / numBlocksPerPeriod) + 1 */
+    /*     uint period; */
+
+    /*     // Price in ETH, scaled by 10**18 */
+    /*     uint priceMantissa; */
+    /* } */
+    function anchors(address asset) external view returns (uint, uint);
+}
+
+
+/**
+ * @notice Price feed conforming to Price Oracle Proxy interface.
+ * @dev Use a single open oracle reporter and anchored to and falling back to the Compound v2 oracle system.
+ * @author Compound Labs, Inc.
+ */
+contract AnchoredView is SymbolConfiguration {
```

Comment:
should AnchoredPriceView.sol be deleted?

Reply:
yeah, may steal that contract name too. I was depending on the old tests passing to see if I broke things, but if this version looks good, will delete the old one